server/migration/1624397093677-AddObservation.ts

```diff
@@ -0,0 +1,18 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddObservation1624397093677 implements MigrationInterface {
+  name = 'AddObservation1624397093677';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`CREATE TABLE "observation_type" ("type" character varying(100) NOT NULL, CONSTRAINT "UQ_45daf0e906ab42b60abeaf83bd5" UNIQUE ("type"), CONSTRAINT "PK_45daf0e906ab42b60abeaf83bd5" PRIMARY KEY ("type"))`);
+    await queryRunner.query(`CREATE TABLE "observation" ("id" SERIAL NOT NULL, "document_id" character varying(100) NOT NULL, "edipi" character varying(10) NOT NULL, "timestamp" character varying(50) NOT NULL, "type" character varying(50) NOT NULL, "unit_id" character varying(50) NOT NULL, "unit" character varying(100) NOT NULL, "org_id" character varying(50) NOT NULL, CONSTRAINT "UQ_c3b96ba3699ceaa1c9c95bdab10" UNIQUE ("id", "document_id"), CONSTRAINT "PK_77a736edc631a400b788ce302cb" PRIMARY KEY ("id"))`);
+    await queryRunner.query(`ALTER TABLE "observation" ADD CONSTRAINT "FK_ea8a73e50440da7b4e5d5893d8b" FOREIGN KEY ("type") REFERENCES "observation_type"("type") ON DELETE RESTRICT ON UPDATE NO ACTION`);
+  }
```

Comment:
If we're sticking with ObservationTypes in a table... should we seed the current ObservationTypes here?

Reply:
Will use report schema instead: https://app.asana.com/0/1198942245525292/1200516880894434/f